### DISCUSSION TITLE
Related data from SQLTrigger?

### DISCUSSION THREAD

**Discussion by gsmith-yprime** [Azure/azure-functions-sql-extension#797] [Q&A] [answered]
I have an Azure function that is using the SqlTrigger binding. Is there any possible configuration that would let me get related data from another table populated in the bound list of SQLChange (sort of like eager loading in EF)? If not, I'm assuming a separate input binding would be the best solution?

**Comment by Charles-Gagnon** [comment, marked as answer]
Currently no, not directly at least. Using an input binding would be an option - although that would require either setting up another function that the one with the sql trigger calls (passing it the appropriate information from the trigger) or using [imperative bindings](https://learn.microsoft.com/en-us/azure/azure-functions/functions-dotnet-class-library?tabs=v4%2Ccmd#binding-at-runtime) for C# functions to dynamically call the input binding.
